packages/cozy-harvest-lib/src/components/Popup.jsx

```diff
@@ -0,0 +1,143 @@
+import { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * customized function to center a popup window
+ * @param  {string} url
+ * @param  {string} title
+ * @param  {string|number} w
+ * @param  {string|number} h
+ * @return {Window}       Popup window
+ */
+// source https://stackoverflow.com/a/16861050
+export function popupCenter(url, title, w, h) {
```

Comment:
this method should only return w,h,top, left. And not creating the popup and so on.

Will work on it